specs/phase0/beacon-chain.md

```diff
@@ -1641,7 +1677,7 @@ def process_randao(state: BeaconState, body: BeaconBlockBody) -> None:
     # Verify RANDAO reveal
     proposer = state.validators[get_beacon_proposer_index(state)]
     signing_root = compute_signing_root(epoch, get_domain(state, DOMAIN_RANDAO))
-    assert bls.Verify(proposer.pubkey, signing_root, body.randao_reveal)
+    assert Eth2Verify(proposer.pubkey, signing_root, body.randao_reveal)
```

Comment:
`Eth2Verify` is not super descriptive (Eth2 verifies a lot of things). What about simply replacing `bls.` with `bls_` to match the style of the Eth2 spec. So `bls.Verify` becomes `bls_verify`.